config/default/core.entity_form_display.node.judgement.default.yml

```diff
@@ -0,0 +1,357 @@
+uuid: d6308cb8-4066-40f5-934b-43d0c2622688
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.judgement.body
+    - field.field.node.judgement.field_case_history
+    - field.field.node.judgement.field_casename
+    - field.field.node.judgement.field_casenumber
```

Comment:
Excel spreadsheet has this as `field_case_number`, perhaps `case_name` too then?

Reply:
I might be looking at the incorrect spreadsheet then, for me it has it as conjoined.

Reply:
It's in the comments field. I'll leave this up to Neil.

Reply:
Ya, i wasn't sure what comments had been approved or not, so went strictly according to the spreadsheet.